addons/event/models/event_mail.py

```diff
@@ -382,43 +387,68 @@ def _prepare_event_mail_values(self):
-  - Scheduled: %(date)s
-  - Template: %(template_name)s (%(template_id)s)
-
-Failed with error:
-  - %(error)s
-
-You receive this email because you are:
-  - the organizer of the event,
-  - or the responsible of the event,
-  - or the last writer of the template.
-""",
-                         event_name=event.name,
-                         event_id=event.id,
-                         date=scheduler.scheduled_date,
-                         template_name=template.name,
-                         template_id=template.id,
-                         error=ex_s)
-                email = self.env['ir.mail_server'].build_email(
-                    email_from=self.env.user.email,
-                    email_to=emails,
-                    subject=subject, body=body,
+    def _warn_error(self, exception):
+        last_error_dt = self.error_datetime
+        now = self.env.cr.now().replace(microsecond=0)
+        if not last_error_dt or last_error_dt < now - relativedelta(hours=1):
+            # message base: event, date
+            event, template = self.event_id, self.template_ref
+            if self.interval_type == "after_sub":
+                scheduled_date = now
+            else:
+                scheduled_date = self.scheduled_date
+            body_content = _(
+                "Communication for %(event_name)s scheduled on %(scheduled_date)s failed.",
+                event_name=event.name,
+                scheduled_date=scheduled_date,
+            )
+
+            # add some information on cause
+            template_link = Markup('<a href="%s">%s (%s)</a>') % (
+                f"{self.get_base_url()}/odoo/{template._name}/{template.id}",
+                template.display_name,
+                template.id,
+            )
+            cause = exception.__cause__
+            exc_qweb = cause and isinstance(cause, QWebException)
+            if exc_qweb:
+                root_cause = cause.__cause__
+                exc_qweb_attr = root_cause and isinstance(root_cause, AttributeError)
+                source_content = _(
+                    "This is due to an error in template %(template_link)s.",
+                    template_link=template_link,
                 )
-                self.env['ir.mail_server'].send_email(email)
-            except Exception as e:
-                _logger.error("Exception while sending traceback by email: %s.\n Original Traceback:\n%s", e, exception)
-                pass
+                if exc_qweb_attr:
+                    error_message = _(
+                        "There is an issue with dynamic placeholder. Actual error received is: %(error)s.",
+                        error=Markup('<br/>%s') % exception_to_unicode(root_cause),
+                    )
+                else:
+                    error_message = _(
+                        "Rendering of template failed with error: %(error)s.",
+                        error=Markup('<br/>%s') % exception_to_unicode(cause),
```

Comment:
Ah, there's special handling for Markup in the translation getter (`get_translation`).. didn't know that. All good then.

Otherwise in principle anything that would format an object to a string turns markup into a plain string, like `'test {}'.format(Markup('<p>')) == 'test <p>'`.